Riot/Modules/Room/DataSources/RoomDataSource.swift

```diff
@@ -197,6 +221,11 @@ extension RoomDataSource {
 
         return editableTextMessage
     }
+    
+    @objc func editableHtmlTextMessage(for event: MXEvent) -> String? {
+        let body: String = event.content["formatted_body"] as? String ?? event.content["body"] as? String ?? ""
+        return body
```

Comment:
Seems we don't have to return an optional here.

```suggestion
    @objc func editableHtmlTextMessage(for event: MXEvent) -> String {
        event.content["formatted_body"] as? String ?? event.content["body"] as? String ?? ""
```